docs-app/app/components/products/product/card.gts

```diff
@@ -0,0 +1,67 @@
+import type { TOC } from '@ember/component/template-only';
+import { hash } from '@ember/helper';
+import { LinkTo } from '@ember/routing';
+import { ContainerQuery, width } from 'ember-container-query';
+
+import type { Product } from '../../../data/products';
+import styles from './card.css';
+import ProductsProductImage from './image';
+
+interface ProductsProductCardSignature {
+  Args: {
+    product: Product;
+    redirectTo?: string;
+  };
+}
+
+const ProductsProductCardComponent: TOC<ProductsProductCardSignature> =
+  <template>
+    <ContainerQuery
+      @features={{hash wide=(width min=320)}}
+      @tagName="article"
+      data-test-product-card
+      class={{styles.container}}
+    >
+      <header class={{styles.header}}>
+        <h2 data-test-field="Name" class={{styles.name}}>
+          {{@product.name}}
+        </h2>
+      </header>
+
+      <div class={{styles.image-container}}>
+        <ProductsProductImage @src={{@product.imageUrl}} />
+      </div>
+
+      <div class={{styles.body}}>
+        <p
+          data-test-field="Short Description"
+          class={{styles.description}}
+        >
+          {{@product.shortDescription}}
+        </p>
+
+        <p data-test-field="Price" class={{styles.price}}>
+          &#36;{{@product.price}}
```

Comment:
The `&#36;` will be shown to the user as `$`. While working on the `docs-app` for `embroider-css-modules`, I had noticed that I couldn't use `$` directly. I'm not sure if this syntax limitation is intended or a bug that temporarily exists.

Reply:
[Chris Krycho explained on Discord](https://discord.com/channels/480462759797063690/1100421354798452789/1100427292523634720):

> This is expected, actually.
> 
> The problem is the intermediate representation is a template literal string.
> 
> Because it's processed into a template literal string as the intermediate representation prior to being handed to Babel, what Babel sees is a string with a super wacky value being interpolated between tokens like this: 
>
> ```js
> INTERMEDIATE = `<p data-test-field="Price" class={{styles.price}}>
>  ${{@product.price}}
> </p>`;
> ```
>
> Notice that that's `${<something>}` which is just normal string interpolation syntax.
>
> We may need to detect that specific string and escape it.
>
> (If anyone is wondering: we would have had the same problem using hbs or really any format; the problem is the intermediate representation using template string literals.)